style.css

```diff
@@ -0,0 +1,222 @@
+html {
+    box-sizing: border-box;
+}
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+body {
+    background-color: #ECEEF1;
+    font-family: AvenirNext,Avenir,"Helvetica Neue",Helvetica,Arial,sans-serif;
+}
+
+#main-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    width: 100vw;
+    height: 10vh;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    color: #505050;
+}
+
+#forgot-pwd {
+    margin-right: 0.5em;
+    text-align: right;
+    line-height: 1.3em;
+}
+
+#signin-link {
+    padding-right: 2em;
+    display: flex;
+    font-size: 12px;
+}
+
+#signin {
+    padding: 0.6em 2em;
+    background-color: #0077C5;
+    border: 1px solid #4C4C4C;
+    border-radius: 3px;
+    font-size: 12px;
+    color: white;
+}
+
+a {
+    text-decoration: none;
+    color: #0077C5;
+    font-size: 12px;
+}
+
+a:visted {
+    color: #0077C5;
+}
+
+#info-link:hover{
+    text-decoration: underline;
+}
+
+#forgot-pwd {
+    display: flex;
+    flex-direction: column;
+}
+
+main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100vw;
+    margin-bottom: 2em;
+}
+
+#header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+#brand {
+    align-self: center;
+    margin-top: 1em;
+}
+
+#links {
+    display: flex;
+    flex-direction: row;
+    margin: 1em;
+}
+
+.brand-links {
+    padding: 0 0.5em;
+}
+
+.svg-brand-link {
+    height: 23px;
+    width: 23px;
+}
+
+#mint-txt {
+    width: 37px;
+    height: 20px;
+}
+
+#qb-txt {
+    width: 98px;
+    height: 22px;
+}
+
+#turbo-txt {
+    width: 74px;
+    height: 22px;
+}
+
+form {
+    background-color: #ffffff;
+    border: 1px solid #c7c7c7;
+    width: 380px;
+    border-radius: 2px;
+}
+
+#form-header {
+    text-align: center;
+    padding: 0 30px 0;
+}
+
+#form-header h2 {
+    font-weight: 400;
+    font-size: 2.4 em;
```

Comment:
The blank space after `2.4` is making the style act funky. Change from `2.4 em` to `2.4em`

Reply:
Removed the space between 2.4 and em